Gemfile

```diff
@@ -67,5 +63,5 @@ gem 'simple_form'
 
 gem 'devise'
 
-
+gem 'pry-rails'
```

Comment:
then can go inside group development & test. No need to have it load in production.